questionnaire/form-helper.js

```diff
@@ -211,6 +215,14 @@ function escapeSchemaContent(schema) {
     return schemaWithEscapedContent;
 }
 
+function shouldShowSignInHyperlink(sectionId) {
+    const uiSchemaValue = uiSchema[sectionId]?.options?.signInHyperlink?.visible;
+    if (uiSchemaValue !== undefined) {
+        return !!uiSchemaValue;
+    }
+    return true;
+}
+
```

Comment:
As discussed, this could perhaps be pulled out in to its on utility with it's own tests. Current tests aren't testing this function.